src/Core/Style.js

```diff
@@ -464,8 +465,6 @@ function _addIcon(icon, domElement, opt) {
  * The first parameter of functions used to set `Style` properties is always an object containing
  * the properties of the features displayed with the current `Style` instance.
  *
- * @property {Number} order - Order of the features that will be associated to
- * the style. It can helps sorting and prioritizing features if needed.
  * @property {Object} fill - Polygons and fillings style.
  * @property {String|Function|THREE.Color} fill.color - Defines the main color of the filling. Can be
  * any [valid color
```

Comment:
can you provide more details about why you remove it please?

Reply:
Currently the style.order was not at all taking into account partialy because it was not called thought getter and setter as all the other properties.
And I also noticed that this properties is only used in VectorTileSource. That's the reason I removed it from style to leave it to the internal use of VectorTile and ordering the different layers inside a same VectorTile. If we would like an ordering of feature inside a same 'layer' we will think at that moment how to put it back into Style.